### PR TITLE
[4.0] Change CSP docs links to Mozilla MDN

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/32163#issuecomment-770134680

### Summary of Changes

Change CSP docs links to Mozilla MDN

### Testing Instructions

Review the strings and check the output in com_csp / plg_system_httpheaders

### Actual result BEFORE applying this Pull Request

URLs pointed to scotthelme.co.uk

### Expected result AFTER applying this Pull Request

URLs point to developer.mozilla.org

### Documentation Changes Required

none

### CODE REVIEW

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on 26d5773e85eb9d51bd926c09f1ddf52f2c59439c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32195">issues.joomla.org/tracker/joomla-cms/32195</a>.</sub>

**Comment by ceford** [comment]
The last link has quoted the German source rather than en-US! Please fix.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32195">issues.joomla.org/tracker/joomla-cms/32195</a>.</sub>

File: administrator/language/en-GB/plg_system_httpheaders.ini [hunk 1]
```diff
@@ -31,7 +31,7 @@ PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_ACTION="Enable default security
...
 PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_TITLE="HTTP Security Headers"
 ; Please do not translate the following two language strings
-PLG_SYSTEM_HTTPHEADERS_REFERRERPOLICY="<a href='https://scotthelme.co.uk/a-new-security-header-referrer-policy/' target='_blank' rel='noopener noreferrer'>Referrer-Policy</a>"
-PLG_SYSTEM_HTTPHEADERS_XFRAMEOPTIONS="<a href='https://scotthelme.co.uk/hardening-your-http-response-headers/#x-frame-options' target='_blank' rel='noopener noreferrer'>X-Frame-Options</a>"
+PLG_SYSTEM_HTTPHEADERS_REFERRERPOLICY="<a href='https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy' target='_blank' rel='noopener noreferrer'>Referrer-Policy</a>"
+PLG_SYSTEM_HTTPHEADERS_XFRAMEOPTIONS="<a href='https://developer.mozilla.org/de/docs/Web/HTTP/Headers/X-Frame-Options' target='_blank' rel='noopener noreferrer'>X-Frame-Options</a>"
```
Comment (zero-24) [inline administrator/language/en-GB/plg_system_httpheaders.ini]:
```suggestion
PLG_SYSTEM_HTTPHEADERS_XFRAMEOPTIONS="<a href='https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options' target='_blank' rel='noopener noreferrer'>X-Frame-Options</a>"
```

**Comment by zero-24** [comment]
Thanks fixed @ceford

**Comment by brianteeman** [comment]
com_csp is still pretty useless #31709 :(

**Comment by zero-24** [comment]
> com_csp is still pretty useless #31709 :(

Is that relevant for this string change? Its on the radar and be taken care of but changing the docs links should not be a problem right?

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on c45f99174554e310b22132050a1de910c76ce819<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32195">issues.joomla.org/tracker/joomla-cms/32195</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c45f99174554e310b22132050a1de910c76ce819<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32195">issues.joomla.org/tracker/joomla-cms/32195</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32195">issues.joomla.org/tracker/joomla-cms/32195</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by zero-24** [comment]
Thanks @bembelimen

**Pull request merged**